igt/Makefile

```diff
@@ -49,7 +49,7 @@ else
 	export EXT_NAME ?=
 endif
 
-CGO ?= 0
+CGO ?= 1
```

Comment:
what's this

Reply:
cgo provides a mechanism for golang and C to call each other which is used in wasm-go-sdk, we must active it.